docs/windows-design.md

```diff
@@ -0,0 +1,261 @@
+# Running Antrea on Windows
+
+Antrea supports running on Windows worker Nodes. In this case, the Windows Node must
```

Comment:
I think no need to repeat "the Windows Node must be a worker", and whether master is Linux is irrelevant to Antrea? Maybe remove the sentence of "In this case...".

Reply:
Sure, I will remove it.